Source/Chromosomes/Vector/gaussian_mutation.cpp

```diff
@@ -2,21 +2,18 @@
 
 #include "gaussian_mutation.h"
 
-gaussian_mutation::gaussian_mutation(const gaussian_mutation_metadata *metadata, stochastic* stochastic_instance) :
-    mutation_internal(metadata) {
-    _mean = metadata->mean;
-    _standard_deviation = metadata->standard_deviation;
-
+gaussian_mutation::gaussian_mutation(const gaussian_mutation_metadata* const metadata, stochastic* stochastic_instance) :
+    _metadata{metadata}, mutation_internal(metadata) {
```

Comment:
For style consistency `mutation_internal` should be initalized with `{ }` not `( )` i.e. `mutation_internal{metadata}`